Sources/Tracing/TracerProtocol.swift

```diff
@@ -323,8 +324,8 @@ extension Tracer {
         }
     }
 
-    /// Start a new ``Span`` and automatically end when the `operation` completes,
-    /// including recording the `error` in case the operation throws.
+    /// Start a new span and automatically end when the operation completes,
```

Comment:
```suggestion
    /// Start a new span and automatically end it when the operation completes,
```

same thing everywhere?